elide-core/src/main/java/com/yahoo/elide/core/datastore/inmemory/InMemoryStoreTransaction.java

```diff
@@ -119,7 +119,7 @@ public Object loadObject(EntityProjection projection,
                 projection.getFilterExpression()) == FeatureSupport.FULL) {
             return tx.loadObject(projection, id, scope);
         } else {
-            return DataStoreTransaction.super.loadObject(projection, id, scope);
+            return super.loadObject(projection, id, scope);
```

Comment:
I think this is wrong.  This was calling the default method defined in the interface. You changed this to call the super class - which is something else - and will cause problems.